Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -484,7 +484,7 @@ public SiteExtensionManager(IContinuousJobsManager continuousJobManager, ITrigge
                                 Executable exe = externalCommandFactory.BuildCommandExecutable(installScript,
                                     installationDirectory,
                                     _settings.GetCommandIdleTimeout(), NullLogger.Instance);
-                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), String.Empty);
+                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), installationArgs ?? String.Empty);
```

Comment:
We have a bunch of functional test in https://github.com/projectkudu/kudu/blob/master/Kudu.FunctionalTests/SiteExtensions/SiteExtensionApiFacts.cs, and we could add coverage for this in one of them.